.travis.yml

```diff
@@ -1,17 +1,26 @@
-sudo: false
+sudo: required
+dist: trusty
 
 language: node_js
 node_js:
   - "6"
 
+addons:
+  chrome: stable
+
 cache:
   yarn: true
 
+before_script:
```

Comment:
Hmm, this shouldn't be needed...

Reply:
You are correct. It should be using the --no-sandbox and --headless mode. It isn't.

Reply:
Possibly hardcoded resolutions forcing an older version of testem that isn't aware of the new syntax (nested `ci` property under `Chrome` section)/

Reply:
I already tried forcing the latest testem. Let me try using an older syntax in testem.js

Reply:
@rwjblue Yay that worked!